CMakeLists.txt

```diff
@@ -200,6 +200,7 @@ option(FLB_IN_NODE_EXPORTER_METRICS    "Enable node exporter metrics input plugi
 option(FLB_IN_WINDOWS_EXPORTER_METRICS "Enable windows exporter metrics input plugin" Yes)
 option(FLB_IN_OPENTELEMETRY            "Enable OpenTelemetry input plugin"            Yes)
 option(FLB_IN_ELASTICSEARCH            "Enable Elasticsearch (Bulk API) input plugin" Yes)
+option(FLB_IN_CALYPTIA_FLEET           "Enable Calyptia Fleet input plugin"           Yes)
```

Comment:
Could you disable to build this plugin on Windows?
We didn't find how to handle reloading on Windows.
To disable this plugin building, could you add the disabling definition in https://github.com/fluent/fluent-bit/blob/master/cmake/windows-setup.cmake?

Reply:
I'm hoping with the last push it will at least compile on windows (waiting on CI). If not I will disable it for now.

Reply:
The windows build failed in CI with this compile error:

```shell
/in_calyptia_fleet.c.obj
in_calyptia_fleet.c
C:\projects\fluent-bit-2e87g\plugins\in_calyptia_fleet\in_calyptia_fleet.c(235): error C2065: 'SIGHUP': undeclared identifier
C:\projects\fluent-bit-2e87g\plugins\in_calyptia_fleet\in_calyptia_fleet.c(390): warning C4020: 'mkdir': too many actual parameters
C:\projects\fluent-bit-2e87g\plugins\in_calyptia_fleet\in_calyptia_fleet.c(397): warning C4020: 'mkdir': too many actual parameters
NMAKE : fatal error U1077: '"C:\Program Files\CMake\bin\cmake.exe"' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\nmake.exe"' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\nmake.exe"' : return code '0x2'
```

Therefore I will disable the plugin on windows for now. Development on the windows support can be added once support for linux and macos are robust.